CachePurgeLibrary/RequestModels/CdnConfiguration.cs

```diff
@@ -11,24 +11,29 @@ namespace CachePurgeLibrary
     public class CdnConfiguration
     {
         public string Hostname { get; set; }
+        
+        public IDictionary<string, bool> PluginIsEnabled { get; set; }
 
-        public IDictionary<string, string> CdnWithCredentials { get; set; }
+        // Credentials are not used for now anyway
+        // public IDictionary<string, string> CdnWithCredentials { get; set; }
```

Comment:
Delete commented out code

Reply:
You're absolutely right, removed those everywhere.